pkg/fuzz/execute.go

```
@@ -122,6 +123,18 @@ func (rule *Rule) Execute(input *ExecuteRuleInput) (err error) {
 				return nil
 			})
 		}
+
+		if rule.options.FuzzStatsDB != nil {
+			component.Iterate(func(key string, value interface{}) error {
```

golangci-lint (1.62.2)

Error return value of `component.Iterate` is not checked

(errcheck)